src/Stratis.Bitcoin.Features.Wallet/WalletSettings.cs

```diff
@@ -66,6 +73,8 @@ public static void BuildDefaultConfigurationFile(StringBuilder builder, Network
             builder.AppendLine("####Wallet Settings####");
             builder.AppendLine("#Save the hex of transactions in the wallet file. Default: 0.");
             builder.AppendLine("#savetrxhex=0");
+            builder.AppendLine("#Creates a default wallet and unlocks the wallet on startup when set to 1. Default: 0.");
```

Comment:
I dont think this actually unlocks the wallet? LoadWallet I think just confirms password but doesnt unlock it if I understand correctly.

Reply:
I think Mike is right @sondreb, `LoadWallet` only loads JSON from disk. You would need to call `CacheSecret` on `WalletTransactionHandler` to unlock the default wallet for further RPC access.

Reply:
I have a solution that unlocks the default wallet, the issue though is that it is not very pretty as it needs some state shared between WalletManager and WalletTransactionHandler. When the PR #3017 from @mikedennis is completed, I will refresh this PR and see how I can make it better.

Reply:
Did a refactoring and improved the unlocking, only supports a single wallet but that is fine, as RPC only supports a single wallet for now.